test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,53 @@
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() throws Exception {
+
+        //empty list
+        assertNoNull();
+
+        //all objects are null
+        assertThereIsNull(null, null);
+        assertThereIsNull(null, null, null);
+
+        //there exists a null object
+        assertThereIsNull(null, "Tom", "Dick", null);
+        assertThereIsNull("Tom", null);
+        assertThereIsNull("Tom", "", "Dick", null);
+        assertThereIsNull("", "Tom", "Dick", null);
+        assertThereIsNull(null, new Integer(1));
+        assertThereIsNull(null, 1, new Integer(1));
+
+        //there is no null objects
+        assertNoNull("Tom", "Dick", "Harry");
+        assertNoNull(1, 2);
+        assertNoNull(1, 2, 3);
+
+        //empty list
+        assertNoNull();
+
+        //all objects are null
+        assertThereIsNull(null, null);
+        assertThereIsNull(null, null, null);
+
+        //there exists a null object
+        assertThereIsNull(null, "Tom", "Dick", null);
+        assertThereIsNull("Tom", null);
+        assertThereIsNull("Tom", "", "Dick", null);
+        assertThereIsNull("", "Tom", "Dick", null);
+        assertThereIsNull(null, new Integer(1));
+        assertThereIsNull(null, 1, new Integer(1));
+
+        //there is no null objects
+        assertNoNull("Tom", "Dick", "Harry");
+        assertNoNull(1, 2);
+        assertNoNull(1, 2, 3);
+            }
+            private void assertThereIsNull(Object... objects) {assertTrue(Utils.isAnyNull(objects));}
```

Comment:
Coding standard violation. `assertTrue(Utils.isAnyNull(objects));` should be at the next line.